### PR TITLE
api hotfix

### PR DESCRIPTION
**Pull request by jinhwak11**

### CODE REVIEW

File: M2/assignment3_m2.py [hunk 1]
```diff
@@ -23,8 +23,14 @@
     id2doc = json.load(f)
 with open('./page_quality_features.json') as f:
     doc2features = json.load(f)
+<<<<<<< HEAD
+with open('./docId_title_map.json') as f:
+    id2title = json.load(f)
+
+=======
```
Comment (nagrawal63) [inline M2/assignment3_m2.py]: Isn't this a remnant of some conflict issue?

File: M2/assignment3_m2.py [hunk 2]
```diff
@@ -23,8 +23,14 @@
...
 with open("./anchor_text_dict.json") as f:
     anchorWordFeatures = json.load(f)
+>>>>>>> 920dfdb8134cc95621c01e2650b1b7a868742433
```
Comment (nagrawal63) [inline M2/assignment3_m2.py]: Same here

**Review by nagrawal63 — COMMENTED** [review]
There are some remnants of your conflict handling here. Looks good apart from that. Feel free to squash and merge after fixing them

**Comment by nagrawal63** [comment]
LGTM!

**Comment by jinhwak11** [comment]
Just give me a second please. I'd like to add time section.

**Comment by nagrawal63** [comment]
Sure, take your time.

**Comment by nagrawal63** [comment]
LGTM! You can squash and merge

**Pull request merged**